eng/common/pipelines/templates/steps/set-default-branch.yml

```diff
@@ -8,4 +8,3 @@ steps:
     echo "##vso[task.setvariable variable=DefaultBranch]$setDefaultBranch"
   displayName: "Setup Default Branch"
   workingDirectory: ${{ parameters.workingDirectory }}
-  condition: eq(variables['DefaultBranch'], '')
```

Comment:
Remove this as eng-sync clone every repo and reset defaultBranch whenever in a new repo.  The condition here will keep the defaultBranch as the same one.

Reply:
True we might need to run this step multiple times in the same pipeline.